peakdet/cli/run.py

```diff
@@ -105,9 +105,10 @@ def get_parser():
     return parser
 
 
+@logger.catch
 def workflow(*, file_template, modality, fs, source='MRI', channel=1,
              output='peakdet.csv', savehistory=True, noedit=False, thresh=0.2,
-             measurements=ATTR_CONV.keys()):
+             measurements=ATTR_CONV.keys(), verbose=True):
```

Comment:
I'd set verbose as False actually - setting the logging to Verbose should be done through CLI.
Also, could you please add "verbose" in the parameters?

Reply:
+1

Reply:
done